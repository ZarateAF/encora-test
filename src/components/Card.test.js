import renderer from "react-test-renderer";
import { Card } from "./";

it("renders correctly", () => {
  const tree = renderer
    .create(<Card  name={"name"}
      stargazerCount={"1000"}
      onClick={a => a}></Card>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
