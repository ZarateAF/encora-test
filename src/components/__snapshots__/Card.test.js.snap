// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
  onClick={[Function]}
>
  <label>
    name
  </label>
  <i
    className="fa-solid fa-star"
  >
    1000
  </i>
</div>
`;
